doc/developers/contributing.rst

```diff
@@ -135,17 +133,16 @@ following rules before submitting:
 How to make a good bug report
 -----------------------------
 
-When you submit an issue to `Github
+When you submit an issue to `GitHub
 <https://github.com/scikit-learn/scikit-learn/issues>`__, please do your best to
 follow these guidelines! This will make it a lot easier to provide you with good
 feedback:
 
 - The ideal bug report contains a :ref:`short reproducible code snippet
-  <minimal_reproducer>`, this way
-  anyone can try to reproduce the bug easily (see `this
-  <https://stackoverflow.com/help/mcve>`_ for more details). If your snippet is
-  longer than around 50 lines, please link to a `gist
-  <https://gist.github.com>`_ or a github repo.
+  <minimal_reproducer>`, this way anyone can try to reproduce the bug easily (see `this
+  <https://stackoverflow.com/help/mcve>`_ for more details). If your snippet is longer
+  than around 50 lines, please link to a `Gist <https://gist.github.com>`_ or a GitHub
+  repo.
```

Comment:
```suggestion
  <minimal_reproducer>`, this way anyone can try to reproduce the bug easily. If your
  snippet is longer than around 50 lines, please link to a 
  `Gist <https://gist.github.com>`_ or a GitHub repo.
```

The Stackoverflow page is already linked from our own internal `minimal_reproducer` page and our page has probably more details so better put emphasis on it.